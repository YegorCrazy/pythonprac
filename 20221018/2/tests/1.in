:sincos x y sin(x)*cos(y)
sincos 1.57 3.14
quit "{}, {}"
