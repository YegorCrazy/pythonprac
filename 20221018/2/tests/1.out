-0.9999984146597762
2, 3
